rider/src/main/kotlin/com/jetbrains/rider/plugins/unity/ui/shaders/ShaderContextPopup.kt

```diff
@@ -0,0 +1,67 @@
+package com.jetbrains.rider.plugins.unity.ui.shaders
+
+import com.intellij.openapi.actionSystem.ActionGroup
+import com.intellij.openapi.actionSystem.DataContext
+import com.intellij.ui.ErrorLabel
+import com.intellij.ui.JBColor
+import com.intellij.ui.components.panels.OpaquePanel
+import com.intellij.ui.popup.PopupFactoryImpl
+import com.intellij.ui.popup.list.PopupListElementRenderer
+import com.intellij.util.FontUtil
+import com.intellij.util.ui.JBUI
+import com.intellij.util.ui.UIUtil
+import icons.UnityIcons
+import java.awt.BorderLayout
+import javax.swing.JComponent
+import javax.swing.JLabel
+import javax.swing.JList
+import javax.swing.JPanel
+
+class ShaderContextPopup(private val group: ActionGroup, private val dataContext: DataContext) :
+    PopupFactoryImpl.ActionGroupPopup("Shader Context", group, dataContext, false, false,
```

Comment:
How about "Use file and symbol context from include at:" to try and make it more obvious what the widget does?

Reply:
Also, is it possible to add a help button to the header? This could go to a wiki page on GitHub with an explanation of what the widget does and why you'd want to change it.

Reply:
`Use file and symbol context from include at:` is too long for header, wiki page is good idea

Reply:
"Include context from:"?